argo-workflow/argo-install.yaml

```diff
@@ -177,15 +185,11 @@ kind: ConfigMap
 metadata:
   name: workflow-controller-configmap
 data:
-  config: | 
```

Comment:
This is a new format of the ConfigMaps https://github.com/argoproj/argo-workflows/blob/master/docs/workflow-controller-configmap.yaml